### ISSUE TITLE
Add Roles based Authentication 

### ISSUE THREAD

**Issue by saroj-17** [kawaaii/Tickbill#6]
You should add a role based authentication where user should register on the system.Only after a registration they can use the system.

**Comment by hridaya2004** [comment]
Only admins can register the users at the moment.

**Comment by hridaya2004** [comment]
I can do as you said, and create a registration panel for Login panel as well. Then admin flips an attribute/flag named as "verified" so that the user can login.

**Comment by saroj-17** [comment]
This seems better. And note that User should have only some view access when they are logged in to the system

**Comment by hridaya2004** [comment]
> User should have only some view access when they are logged in to the system

This is already done as per commit e8c15feace9c01b2fbf8fc3fe606bff12c51abaa.
